packages/core/src/accordion/accordion-panel.element.ts

```diff
@@ -40,11 +49,22 @@ import { styles } from './accordion-panel.element.css.js';
  * @event expandedChange - notify when the user has clicked the panel header
  * @cssprop --border-color
  * @cssprop --border-width
- * @cssprop --color
- * @cssprop --font-size
- * @cssprop --font-weight
+ * @cssprop --animation-duratino
+ * @cssprop --animation-easing
  */
-export class CdsAccordionPanel extends LitElement {
+@animate({
+  expanded: {
+    true: AnimationAccordionPanelOpenName,
+    false: reverseAnimation(AnimationAccordionPanelOpenName),
+  },
+})
+export class CdsAccordionPanel extends LitElement implements Animatable {
+  @property({ type: String })
+  cdsMotion = 'on';
+
+  @event()
+  motionChange: EventEmitter<string>;
```

Comment:
Since motion is a common API across components and we prefix it like `cds-layout`, `cds-theme` and `cds-i18n` seems like the event should correspond as well `cdsMotion` and `cdsMotionChange`.

Reply:
Yeah, that makes sense. I can do that.